Gemfile

```diff
@@ -27,6 +27,8 @@ gem "simplecov-rcov"
 gem "toastr-rails"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 5.0"
+gem "ransack"
+gem "axlsx_rails"
```

Comment:
gem này hỗ trợ những cái gì

Reply:
Gem hỗ trợ
- Export ra các bản báo cáo dạng xlsx
- Format các loại trong file (màu sắc, định dạng, kiểu chữ, cỡ chữ, căn lề,...)
- Tự động xác định kiểu dữ liệu đang dùng
- Hỗ trợ tính toán, gộp hàng cột
- Tùy chỉnh headẻr, footer
- Tạo biểu đồ từ database trong worksheet